### PR TITLE
New top-level navigation on AGP

### PR DESCRIPTION
**Pull request by Sandravaphilips**
- Issue https://github.com/laws-africa/peachjam/issues/909
- Loom https://www.loom.com/share/ae3fb5c4b1584084a65b628d31a88b1c
- Navigation didn't look good on screens from 992px to around 1030px so I increased the max width of screens over 992px by 10px. I can revert changes if you think it is unnecessary.

Before the change:
Screenshot 2023-02-16 at 14 05 52

After the change on the same width as above:
Screenshot 2023-02-16 at 14 06 12

### CODE REVIEW

**Comment by Sandravaphilips** [comment]
https://www.loom.com/share/4ed0e1c793f041518075bd8a467a766a

File: peachjam/static/bootstrap/scss/_variables.scss [hunk 1]
```diff
@@ -486,7 +486,7 @@ $grid-breakpoints: (
 $container-max-widths: (
   sm: 540px,
   md: 720px,
-  lg: 960px,
+  lg: 970px,
```
Comment (longhotsummer) [inline peachjam/static/bootstrap/scss/_variables.scss:489]: This is going to get lost next time we upgrade bootstrap.

Is it critical we have this?

File: africanlii/templates/peachjam/_header.html [hunk 2]
```diff
@@ -1,45 +1,125 @@
...
+          px-0
+          {%if view.navbar_link == 'legal_instruments'%}
+          active
+          {%endif%}
+        "
+        href="{% url 'legal_instrument_list' %}"
+        >
+        {% trans 'Legal Instruments' %}
+
+        <div class="dropdown-summary text-muted">Charters, treaties, conventions and regulations.</div>
```
Comment (longhotsummer) [inline africanlii/templates/peachjam/_header.html:55]: Please mark the summary strings as translatable.

File: peachjam/static/stylesheets/components/_navbar.scss [hunk 3]
```diff
@@ -9,3 +12,23 @@
     }
   }
 }
+
+.dropdown-summary {
+  font-size: smaller;
+}
+
+.dropdown-menu {
+  border: none;
```
Comment (longhotsummer) [inline peachjam/static/stylesheets/components/_navbar.scss:21]: I think we should keep the border so it's in keeping with our UI patterns.

**Review by longhotsummer — COMMENTED** [review]
Looks great, minor changes :)

File: peachjam/static/bootstrap/scss/_variables.scss (hunk 1 above)
Comment (Sandravaphilips) [reply peachjam/static/bootstrap/scss/_variables.scss:489]: No. it's not

**Comment by Sandravaphilips** [comment]
I forgot I was working on an incorrect branch and pushed my changes. This is how the dropdown looks now though:
Screenshot 2023-02-20 at 08 35 05

**Review by longhotsummer — APPROVED** [review]

**Pull request merged**